src/utils/utils_windows_common.c

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#include <windows.h>
+
```

Comment:
remove redundant empty line and sort these a-z

Reply:
This empty line is not redundant. `#include <windows.h>` has to be the first header inluded. If I remove it, the coding style will sort the includes and it will stop building.

Reply:
wow, ok, I didn't know that - thx